src/components/CanvasSaveButton/CanvasSaveButton.js

```diff
@@ -256,7 +321,9 @@ class CanvasSaveButton extends PureComponent {
         );
         destContext.restore();
 
-        resolve();
+        // Return the pixels width of the arrow and the margin right,
+        // that must not be occupied by the copyright.
+        resolve(clip.w - (clip.w - 2 * this.padding - arrowWidth));
```

Comment:
seems equivalent  to  
```suggestion
        resolve(arrowWidth + 2 * this.padding);
```